src/posthog-core.ts

```diff
@@ -161,6 +161,7 @@ const defaultConfig = (): PostHogConfig => ({
     name: 'posthog',
     callback_fn: 'posthog._jsc',
     bootstrap: {},
+    disable_compression: false,
```

Comment:
should this be `enable_compression` and default to true 🤷‍♀️

Reply:
no, the irregularity fits this library :D